src/content_handlers/script_handler/CMakeLists.txt

```diff
@@ -31,7 +31,8 @@ target_link_libraries (
             aduc::system_utils
             aduc::workflow_data_utils
             aduc::workflow_utils
-            -zdefs
-            )
+            -zdefs)
+
+target_compile_definitions (${target_name} PUBLIC ADUSHELL_FILE_PATH="${ADUSHELL_FILE_PATH}")
```

Comment:
nit: Was there are reason it is PUBLIC? I think PRIVATE should be sufficient and is more consistent with others.